Developers/Git_Basics.rst

```diff
@@ -149,7 +149,7 @@ parts. They should have a newline between them.
 .. seealso::
```

Comment:
please enclose Writing Good Commits in double backticks as i did (but cant be seen due to github UI)